README.md

```diff
@@ -46,7 +46,10 @@ Debian/Ubuntu/Mint:    `$ sudo apt-get install build-essential libxmu-dev libxi-
 RedHat/CentOS/Fedora:  `$ sudo yum install libXmu-devel libXi-devel libGL-devel git`
 
 ##### Build
-
+	#For Unix/Mac systems (see bellow for more information)
```

Comment:
If you're willing to leave the README.md changes out, I'll go ahead and merge the rest.